PyRoute/Star.py

```diff
@@ -729,12 +756,15 @@ def calculate_army(self):
             self.im_be = 0
 
     def _ehex_to_int(self, value):
+        return Utilities.ehex_to_int(value)
+
         val = int(value, 36) if value in '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ' else 0
```

Comment:
I think this is now dead code can can be removed.

Reply:
Cleaned this up after you commented